modules/cluster-logging-viewing-logs-console.adoc

```diff
@@ -3,27 +3,28 @@
 // * logging/cluster-logging-viewing.adoc
 
 [id="cluster-logging-viewing-logs-console_{context}"]
-= Viewing cluster logs in the {product-title} web console 
+= Viewing pod logs in the web console 
 
-You can view cluster logs in the {product-title} web console . 
+You can view the pod log for various resources in the {product-title} web console. 
 
-.Prerequisites
-
-* OpenShift Logging and Elasticsearch must be installed.
+[NOTE]
+====
+Some resources, such as builds, do not have pods to query directly. Locate the *Logs* link on the *Details* page.
```

Comment:
I find this to contradict: https://github.com/openshift/openshift-docs/pull/30366/files#diff-7430c8818e8d42b91e9a82a6614f517c68e1c5d36b58d0fc4d72010d7e11a510R10 

Can we elaborate more on what this means? The pods are no longer 'available'/'exist' their for we can't pull logs from them?

Reply:
@sferich888 
I was trying to say that, while the procedure tells users to select a pod and click *Logs*, some resources don't have pods associated with them, but you can still get logging info for those resources.